prog/mutation.go

```diff
@@ -374,6 +374,30 @@ func (t *BufferType) mutate(r *randGen, s *state, arg Arg, ctx ArgCtx) (calls []
 	case BufferText:
 		data := append([]byte{}, a.Data()...)
 		a.data = r.mutateText(t.Text, data)
+	case BufferCompressed:
+		data := a.Data()
+		if len(data) == 0 {
+			return
+		}
+		data, err := Decompress(data)
+		if err != nil || len(data) == 0 {
+			return // Leave data unchanged.
+		}
+		hm := MakeGenericHeatmap(data)
+		// At least one mutation, up to about one mutation every 256 KB
+		numMutations := r.Intn(len(data)/(1<<18)+1) + 1
+		for i := 0; i < numMutations; i++ {
+			index := hm.ChooseLocation(r.Rand)
+			width := 1 << uint(r.Intn(4))
+			if index+width <= len(data) {
+				storeInt(data[index:], r.Uint64(), width)
+			}
+		}
+		data, err = Compress(data)
+		if err != nil {
+			return // Leave data unchanged.
```

Comment:
Silent ignoring of errors is not good. If it will start happening in production, we will never know. We don't expect this to fail at all since it compresses to an in-memory buffer, right? I would make Compress panic on errors internally and don't return any errors at all.

Reply:
I will also apply the same reasoning to encoding to base64 (`EncodeB64`)